build.gradle

```diff
@@ -2,11 +2,15 @@
 buildscript {
   repositories {
     jcenter()
+    maven {
+      url 'http://oss.jfrog.org/artifactory/plugins-release'
+    }
```

Comment:
Just found that this plugin is also available for the plugins element: https://plugins.gradle.org/plugin/com.jfrog.artifactory
The documentation seems to lag behind. Will try that.